jwst/associations/lib/rules_level3_base.py

```diff
@@ -840,3 +897,27 @@ def _init_hook(self, item):
 
         self.data['asn_type'] = 'spec3'
         super(AsnMixin_Spectrum, self)._init_hook(item)
+
+class AsnMixin_AuxData:
+    """Process special and non-science exposures as science.
+    """
+    def get_exposure_type(self, item, default='science'):
+        """Override to force exposure type to always be science
+        Parameters
+        ----------
+        item : dict
+            The pool entry to determine the exposure type of
```

Comment:
Ack! Dangling participle (or something like that). How about "The pool entry for which the exposure type is determined".

Reply:
Updated the text. I also missed adding the diff of the truth file and the output file so for completeness,
diffs output_pool/jw00623-o003_spec3_001_asn.json /Users/ddavis/data/JWST/CTeam/associations/jw00623-o003_spec3_001_asn.json
    "asn_rule": "candidate_Asn_Lv3Spec",		      |	    "asn_rule": "candidate_Asn_IFU",
    "code_version": "0.13.2a.dev102+g3f32b46e",		      |	    "code_version": "0.13.2a.dev153",
    "constraints": "DMSAttrConstraint({'name': 'program', 'so |	    "constraints": "DMSAttrConstraint({'name': 'program', 'so
            "name": "jw00623-o003_t002_miri_clear",	      |	            "name": "jw00623-o003_t002_miri",

>
So only the asn rule name, the code version and the list of constraints are different.